test/tests/functional/pbs_power_provisioning_sgi.py

```diff
@@ -277,6 +285,107 @@ def NodesetDelete( nodeset_name ):
                 jid = self.eoe_job(x, eoe)
                 self.energy_check(jid)
 
+    def test_sgi_eoe_job(self):
+        """
+        Submit jobs with an eoe values and check that messages are logged
+        indicating PMI activity, and current_eoe and resources_used.energy
+        get set.
+        """
+        eoes = ['100W', '150W', '450W']
+        while len(eoes) > 0:
+            eoe_profile = eoes.pop()
+        for x in range(1, self.power_nodes + 1):
+            jid = self.eoe_job(x, eoe_profile)
+            self.energy_check(jid)
+
+    def test_sgi_request_more_eoe(self):
+        """
+        Submit jobs with available+1 eoe chunks and verify job comment.
```

Comment:
I'm not sure what you are testing here is power related at all. The scheduler will probably bail out before it even starts looking at the eoe resource request. I'm not saying it is an invalid test, but it is not testing what you want it to test.

Reply:
@arungrover  In this test case, submitting a job which is requesting more power nodes so in such case job will be queued with appropriate comment. Am i getting wrong comment ?

Reply:
Well, if a job asks for more nodes than available then it will get that message. The test has nothing to do with power feature. scheduler just matches eoe as any other resource. But it is okay even if you have the test in place.